charts/rancher-logging/0.2.5/charts/fluentd/charts/fluentd-linux/Chart.yaml

```diff
@@ -0,0 +1,17 @@
+name: fluentd-linux
+version: 0.0.2
+appVersion: 1.6.3
```

Comment:
Don't all of these sub-chart versions have to change too ? https://github.com/rancher/system-charts/blob/dev-v2.5/charts/rancher-logging/0.2.4/charts/fluentd/charts/fluentd-linux/Chart.yaml#L2-L3

Reply:
it looks like we don't usally upgrade them.  I imagine helm calls upgrade on them given they are packaged with the parent chart and they get upgraded.

Reply:
As a helm best practice it should be updated because sub charts should be standalone.  Given we only use the sub charts here in this chart it doesn't affect anything.  This is something we may want to do going forward though.